### ISSUE TITLE
publish play-workshop module 1.0.0

### ISSUE THREAD

**Issue by serapath** [ethereum/play#128]
## `@todo`
* [x] [#15 support chat integration](https://github.com/ethereum/play/issues/15)
* [x] refactor `play-workshop` repo
* [ ] [#64 fix workshop description markdown parser](https://github.com/ethereum/play/issues/64)
* [x] add split view for no-tool lessons
* [x] add default gitter chat if no support chat is given
* [ ] if no `info` field, then show CHAT and info not at all
* [ ] fix chat size
* [ ] ERROR - font https://ninabreznik.github.io/play-workshop-chainsafe-blockchainbootcamp-introduction/
* [ ] font good (you just have to be on the internet)
* [ ] fix keyboard navigation - workshop - when you click back/forward with left/right arrows it goes to 0, -1, -2, -3
* [ ] prevent taking focus of an iframe (example Remix)
* [ ] markdown - errors in code snippets from markdown
* [ ] make skilltree dropdown themeable/styleable
* [ ] make skilltree onhover infobox "nicer" (e.g. adapt height to content)
* [ ] fix font - font not good (see https://ninabreznik.github.io/play-workshop-chainsafe-blockchainbootcamp-introduction-to-solidity/)
* [x] links in skilltree are to the workshop.json, not the page itself
* [ ] enable `workshop.defaults.css` and `workshop.defaults.config`
* [ ] add in `style()` that it actually overwrites `"csjs classes"` with passed in custom `css`
* [ ] improve themable "menubar" on top of the tool
* [ ] [#127 add meta workshop (workshop-workshop)](https://github.com/ethereum/play/issues/127)
* [ ] rename theme colors
* [ ] integrate url registry into workshop module
* [ ] update markdown module (to display special characters correctly (>^ etc))
* [ ] fix asset imports
* [ ] [#55 refine & theme workshop module (+workshop.json format)](https://github.com/ethereum/play/issues/55)
* [ ] make assets (images, fonts, etc...) nicely "importable"
* [ ] make responsive on all resolutions and fix styling
* [ ] make nicely themeable and updateable at runtime with
  * e.g. `.postMessage`
  * e.g. --foo-bar: red;
  * e.g. background-color: var(--foo-bar);
* [ ] make hyper modular
* [ ] make it work with/without iframe
* [ ] make it use `twm`
* [ ] feature: load from url
* [ ] feature: load missing from localstorage
* [ ] feature: save custom from url2localstorage
* [ ] feature: load missing from defaults
* [ ] feature: update url with custom missing
* [ ] make generic workshopping to
  * load `workshop.json` path from url
  * check `workshopping@vX.X.X` version for compatibility
  * maybe add `version` to `workshop.json`
* [ ] think about how to notify users about workshop updates
* [ ] feature: `workshop.json#contribute` field
* [ ] feature: wallet address
    * maybe each lesson and "need/dependency" and "unlocks/options" item/link might have a wallet address
    * The goal would be to find an "address convention", e.g.
    * ```
      type: e.g. "ethereum mainnet"
      address: 0x12345678
      "shares": e.g. 0.25 or 100000

      Summing up all shares and address listed (if no number for "shares" is given for an address,
      it would just split the remaining amount equally to all of those shareless addresses)

      e.g.
      1. learner finishes task with bounty `200`
      2. workshop says `"share 10%"` => so learner receives 180 and 20 go to maintainers
      3. 10% from that (=2) will be split amongst addresses based on their shares/totalShares
      ```
    * maybe address convention:  `0x123456789@main.ethereum.net?share=15,0x987654321@main.ethereum.net?share=23`
    * inspiration: https://www.fastmail.com/help/receive/addressing.html

---
